ui/app/src/components/HistoryDialog/HistoryDialogContainer.tsx

```diff
@@ -155,15 +155,30 @@ export function HistoryDialogContainer(props: HistoryDialogContainerProps) {
     } else {
       fetchContentByCommitId(site, item.path, version.versionNumber).subscribe((content) => {
         const image = isImage(item);
-        dispatch(
-          showPreviewDialog({
-            type: image ? 'image' : 'editor',
-            title: item.label,
-            [image ? 'url' : 'content']: content,
-            mode: image ? void 0 : getEditorMode(item),
-            subtitle: `v.${version.versionNumber}`
-          })
-        );
+
+        if (content instanceof Blob) {
```

Comment:
I though it would be good to have this validation here (even though I added fonts to the fetchContentByCommitId service), just in case there's a non-handled Blob.